docs/services/motion/_index.md

```diff
@@ -0,0 +1,466 @@
+  - **Transforms**: These are a list of `PoseInFrame` messages that specify arbitrary other transformations that will be ephemerally added to the frame system at solve time.
+
+- `constraints` ([Constraints](https://pkg.go.dev/go.viam.com/api/service/motion/v1#Constraints)): Pass in optional [motion constraints](./constraints/).
+  By default, motion is unconstrained with the exception of obstacle avoidance.
+
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): A generic struct, containing extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(bool)](https://pkg.go.dev/builtin#bool): Whether the move was successful.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/motion#Service).
+
+**Example usage:**
+
+```go {class="line-numbers linkable-line-numbers"}
+ // Access the Motion Service
+motionService, err := motion.FromRobot(robot, "builtin")
+if err != nil {
+  logger.Fatal(err)
+}
+
+// Assumes a gripper configured with name "my_gripper" on the robot
+myFrame := "my_gripper_offset"
+
+goalPose := PoseInFrame(0, 0, 300, 0, 0, 1, 0)
+
+// Move the gripper
+moved, err := motionService.Move(context.TODO(), goalPose, worldState, nil, nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### MoveSingleComponent
+
+*`MoveSingleComponent` allows you to bypass Viam’s internal motion planning entirely, if desired, for a single component.*
+
+`MoveSingleComponent` looks similar to `Move` above but may result in radically different behavior.
+
+`MoveSingleComponent` translates the given destination into the frame of the specified component, and then calls `MoveToPosition` on that component to move it to the desired location.
+As of April 21, 2023, [arms](/components/arm/) are the only component supported by `MoveSingleComponent`.
+
+As the name of the method suggests, only the single component specified by `component_name` will actuate.
+
+An example of when this may be useful is if you have implemented your own custom arm model, and wish to use your own motion planning for it.
+Implement `MoveToPosition` on that arm using whatever method you desire to plan motion to the specified pose, and then use `MoveSingleComponent` to pass the destination in the frame of any other robot component.
+
+{{% alert title="Note" color="note" %}} <a id="move-vs-movetoposition">
+
+If this method is called with an arm that uses Viam’s motion planning on the backend, then this method is equivalent to using `robot.TransformPose` to transform the destination into the frame of the arm, and then calling [`MoveToPosition`](/components/arm/#movetoposition) on the arm directly.
+Note that `arm.MoveToPosition` does not use `world_state`, so collision checking and obstacle avoidance *will not* be performed.
```

Comment:
I do not think the arm docs need a note because if someone is using the arm API instead of the motion API, they would not expect that the motion service is coming into play. The arm API doesn't have a world_state parameter; this note is only here because this motion service method DOES have a world_state parameter but it might not be used when the call is executed.

Reply:
Put another way, if someone is looking at the arm docs, none of the methods have a world_state so the reader would never think a world_state would be used. If someone is looking at this doc, they wouldn't necessarily realize that this motion service method actually indirectly calls an arm method.

Reply:
Peter suggests that we add a note to arm saying that one arm can hit another arm if you're just using the arm API, and that to avoid that you should use the motion service, but that's not specific to this section or to this PR.

Reply:
I'm thinking if someone uses the arm component but doesn't yet know about the motion service it may be nice? But maybe calling out to it at the end which I think we do is enough

Reply:
Yeah it's just that we would link to either the motion service in general, or to the `move` method, because this movesinglecomponent method does NOT really employ the motion service as expected so we wouldn't want to link them to here.